package.json

```diff
@@ -11,11 +12,8 @@
     "test:watch": "jest --watch",
     "prepare": "husky install",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
-  },
-  "engines": {
```

Comment:
Do we have to remove it because otherwise deploy on heroku doesnt work, or is there another reason? 🤔😁

Reply:
I wanted to ask you if thats really necessary cause heroku cannot execute 'npm please-use-yarn' command. I assume its kind of a fake thing? If we need it i will try to find another solution but it was quite hard to find any... Anyway heroku use yarn during the deployment because it detects the yarn lock file

Reply:
For me, we can delete it if it casues problems on the heroku. Maybe I will find another way to force use of yarn instead of npm, or we just have to remember that we are using yarn 😅

Reply:
Heroku can use yarn instead of npm, so we can preserve those lines. As I remember the only thing you need to do is to replace `npm run start:production` with `yarn start:production`

Reply:
@KonradSzwarc Yeah, you're right, now it works just fine :)

Reply:
Oh, no, I said it too fast, it failed :P